client/src/modules/Registry/components/PersonalInfo/index.test.tsx

```diff
@@ -0,0 +1,180 @@
+  });
+
+  test('should render form items with proper values', async () => {
+    setup();
+
+    const firstName = await screen.findByDisplayValue(mockValues.firstName);
+    const lastName = await screen.findByDisplayValue(mockValues.lastName);
+    const primaryEmail = await screen.findByDisplayValue(mockValues.primaryEmail);
+    const contactsEpamEmail = await screen.findByDisplayValue(mockValues.contactsEpamEmail);
+
+    expect(firstName).toBeInTheDocument();
+    expect(lastName).toBeInTheDocument();
+    expect(primaryEmail).toBeInTheDocument();
+    expect(contactsEpamEmail).toBeInTheDocument();
+  });
+
+  test.each`
+    label
+    ${LABELS.firstName}
+    ${LABELS.lastName}
+    ${LABELS.primaryEmail}
+    ${LABELS.epamEmail}
+  `('should render field with $label label', async ({ label }) => {
+    setup();
+    const fieldLabel = await screen.findByLabelText(label);
+    expect(fieldLabel).toBeInTheDocument();
+  });
+
+  test('should render field with location label', async () => {
+    setup();
+    const fieldLabel = await screen.findByText(LABELS.location);
+    expect(fieldLabel).toBeInTheDocument();
+  });
+
+  test.each`
+    placeholder
+    ${PLACEHOLDERS.firstName}
+    ${PLACEHOLDERS.lastName}
+    ${PLACEHOLDERS.email}
+    ${PLACEHOLDERS.epamEmail}
+  `('should render field with $placeholder placeholder', async ({ placeholder }) => {
+    setup();
+    const fieldPlaceholder = await screen.findByPlaceholderText(placeholder);
+    expect(fieldPlaceholder).toBeInTheDocument();
+  });
+
+  test('should render error message on wrong email input', async () => {
+    setup();
+
+    const wrongEmail = 'test';
+
+    const emailInput = await screen.findByPlaceholderText(PLACEHOLDERS.email);
+    const noErrorMessage = screen.queryByText(ERROR_MESSAGES.email);
+    expect(emailInput).toBeInTheDocument();
+    expect(noErrorMessage).not.toBeInTheDocument();
+
+    fireEvent.change(emailInput, {
+      target: {
+        value: wrongEmail,
+      },
+    });
+
+    expect(emailInput).toHaveValue(wrongEmail);
+
+    const errorMessage = await screen.findByText(ERROR_MESSAGES.email);
+    expect(errorMessage).toBeInTheDocument();
+  });
+
+  test('should render error message on wrong EPAM email input', async () => {
+    setup();
+
+    const wrongEmail = 'test@test.com';
+
+    const emailInput = await screen.findByPlaceholderText(PLACEHOLDERS.epamEmail);
+    const noErrorMessage = screen.queryByText(ERROR_MESSAGES.epamEmail);
+    expect(emailInput).toBeInTheDocument();
+    expect(noErrorMessage).not.toBeInTheDocument();
+
+    fireEvent.change(emailInput, {
+      target: {
+        value: wrongEmail,
+      },
+    });
+
+    expect(emailInput).toHaveValue(wrongEmail);
+
+    const errorMessage = await screen.findByText(ERROR_MESSAGES.epamEmail);
+    expect(errorMessage).toBeInTheDocument();
+  });
+
+  test('should render error message on wrong first name input', async () => {
+    setup();
+
+    const wrongName = 'Джон';
```

Comment:
use English text pls

Reply:
end up using polish, don't know why `englishNamePattern` includes numbers, maybe I should add another english pattern that allows only letters?

Reply:
For this case it's ok